packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/get-record-locator.test.ts

```diff
@@ -0,0 +1,303 @@
+import test, { Test } from "tape-promise/tape";
+import { v4 as uuidv4 } from "uuid";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import {
+  EthContractInvocationType,
+  Web3SigningCredentialType,
+  PluginLedgerConnectorBesu,
+  PluginFactoryLedgerConnector,
+  ReceiptType,
+  InvokeContractV1Request,
+} from "../../../../../main/typescript/public-api";
+import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
+import {
+  BesuTestLedger,
+  pruneDockerAllIfGithubAction,
+} from "@hyperledger/cactus-test-tooling";
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+import HelloWorldContractJson from "../../../../solidity/hello-world-contract/HelloWorld.json";
+import Web3 from "web3";
+import { PluginImportType } from "@hyperledger/cactus-core-api";
+
+const testCase = "get record locator";
+const logLevel: LogLevelDesc = "TRACE";
+
+test("BEFORE " + testCase, async (t: Test) => {
+  const pruning = pruneDockerAllIfGithubAction({ logLevel });
+  await t.doesNotReject(pruning, "Pruning didn't throw OK");
+  t.end();
+});
+
+test(testCase, async (t: Test) => {
+  const besuTestLedger = new BesuTestLedger();
+  await besuTestLedger.start();
+
+  test.onFinish(async () => {
+    await besuTestLedger.stop();
+    await besuTestLedger.destroy();
+  });
+
+  const rpcApiHttpHost = await besuTestLedger.getRpcApiHttpHost();
+  const rpcApiWsHost = await besuTestLedger.getRpcApiWsHost();
+
+  /**
+   * Constant defining the standard 'dev' Besu genesis.json contents.
+   *
+   * @see https://github.com/hyperledger/besu/blob/1.5.1/config/src/main/resources/dev.json
+   */
+  const firstHighNetWorthAccount = "627306090abaB3A6e1400e9345bC60c78a8BEf57";
+  const besuKeyPair = {
+    privateKey:
+      "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+  };
```

Comment:
@AzaharaC We have a newer, less copy-pasty way of getting the high net worth genesis account (same Priv/Pub keys as here but obtained from the besu test ledger directly).

```suggestion
  const firstHighNetWorthAccount = besuTestLedger.getGenesisAccountPubKey();
  const besuKeyPair = {
    privateKey: besuTestLedger.getGenesisAccountPrivKey(),
  };
```